webapp/admin/views.py

```diff
@@ -226,3 +230,133 @@ def update_invite_status(store_id):
             flask.flash(msg, "negative")
 
     return jsonify(res)
+
+
+# ---------------------- MODELS SERVICES ----------------------
+@admin.route("/admin/store/<store_id>/models")
+@login_required
+def get_models(store_id):
+    """
+    Retrieves models associated with a given store ID.
+
+    Args:
+        store_id (int): The ID of the store for which to retrieve models.
+
+    Returns:
+        dict: A dictionary containing the response message, success status,
+        and data.
+    """
+    res = {}
+    try:
+        models = admin_api.get_store_models(flask.session, store_id)
+        res["message"] = "Models retrieved"
```

Comment:
If the request is successful we don't need a message :)

Reply:
For all requests? Or for the get requests only?

Reply:
All successful requests